managed/models/settings_helpers.go

```diff
@@ -137,6 +144,17 @@ func UpdateSettings(q reform.DBTX, params *ChangeSettingsParams) (*Settings, err
 		return nil, NewInvalidArgumentError(err.Error())
 	}
 
+	if params.DefaultRoleID != 0 {
+		tx, ok := q.(*reform.TX)
+		if !ok {
+			return nil, fmt.Errorf("%w: changing Role ID requires a *reform.TX", ErrTxRequired)
+		}
```

Comment:
Bad thing about this that it in fact it converts compile time error into runtime error. I think we should discuss our approaches, and maybe replace `reform.DBTX`, `reform.Querier` to `reform.TX`. That will force devs always to wrap db operations in transaction and will remove this types problems.

Reply:
Agreed. The downside of transactions is a slower performance though.

Reply:
There is no performance issues with this DB at all, we have too few requests.